wpt-metadata/chromedriver/headless/webdriver/tests/bidi/network/response_started/response_started_cached.py.ini

```diff
@@ -1,4 +1,5 @@
 [response_started_cached.py]
+  expected: TIMEOUT
```

Comment:
Do you know why the tests started timing out?

Reply:
Time out is better than the status quo: 
<img width="1167" alt="image" src="https://github.com/GoogleChromeLabs/chromium-bidi/assets/2840106/98007b65-e2ff-4af4-9cd9-9710741ac94a">

But no, I don't know